src/components/Service/Tappable/Tappable.tsx

```diff
@@ -36,8 +48,8 @@ export const Tappable = forwardRef(({
         interactiveAnimation === 'opacity' && styles['wrapper--opacity'],
         className,
       )}
-      onPointerCancel={onPointerCancel}
-      onPointerDown={onPointerDown}
+      onPointerCancel={handlePointerCancel}
```

Comment:
Thank you for your issue and contribution!
It would be preferred to use function `callMultiple`

Reply:
@mainsmirnov yep, good one